output_results.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"strings"
+	"io"
+)
+
+// RunOutputTask will read in the analysis.json file and output
+// any useful information to o or any errors to e.
+func RunOutputTask(o, e io.Writer, runner Runner, resourceFactory DumpedJSONResourceFactory, workers int) {
```

Comment:
Why do we write it to 2 different writers?

Seems that so far `e` is never used?